Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -303,65 +277,29 @@ public final class SinglePromptSecureEnclaveValet: NSObject {
 
 @available(macOS 10.11, *)
 extension SinglePromptSecureEnclaveValet {
-    
+
     // MARK: Public Class Methods
-    
-    /// - parameter identifier: A non-empty string that uniquely identifies a SinglePromptSecureEnclaveValet.
-    /// - returns: A SinglePromptSecureEnclaveValet that reads/writes keychain elements with the desired flavor.
+
+    /// - Parameter identifier: A non-empty string that uniquely identifies a SinglePromptSecureEnclaveValet.
+    /// - Returns: A SinglePromptSecureEnclaveValet that reads/writes keychain elements with the desired flavor.
     @objc(valetWithIdentifier:accessControl:)
     public class func 🚫swift_valet(with identifier: String, accessControl: SecureEnclaveAccessControl) -> SinglePromptSecureEnclaveValet? {
         guard let identifier = Identifier(nonEmpty: identifier) else {
             return nil
         }
-        
+
         return valet(with: identifier, accessControl: accessControl)
     }
-    
-    /// - parameter identifier: A non-empty string that must correspond with the value for keychain-access-groups in your Entitlements file.
-    /// - returns: A SinglePromptSecureEnclaveValet that reads/writes keychain elements that can be shared across applications written by the same development team.
+
+    /// - Parameter identifier: A non-empty string that must correspond with the value for keychain-access-groups in your Entitlements file.
+    /// - Returns: A SinglePromptSecureEnclaveValet that reads/writes keychain elements that can be shared across applications written by the same development team.
     @objc(sharedAccessGroupValetWithIdentifier:accessControl:)
     public class func 🚫swift_sharedAccessGroupValet(with identifier: String, accessControl: SecureEnclaveAccessControl) -> SinglePromptSecureEnclaveValet? {
         guard let identifier = Identifier(nonEmpty: identifier) else {
             return nil
         }
         return sharedAccessGroupValet(with: identifier, accessControl: accessControl)
     }
-    
-    // MARK: Public Methods
-    
-    /// - parameter key: A Key used to retrieve the desired object from the keychain.
-    /// - parameter userPrompt: The prompt displayed to the user in Apple's Face ID, Touch ID, or passcode entry UI.
-    /// - returns: The data currently stored in the keychain for the provided key. Returns `nil` if no object exists in the keychain for the specified key, or if the keychain is inaccessible.
-    @available(swift, obsoleted: 1.0)
-    @objc(objectForKey:userPrompt:userCancelled:)
-    public func 🚫swift_object(forKey key: String, withPrompt userPrompt: String, userCancelled: UnsafeMutablePointer<ObjCBool>?) -> Data? {
```

Comment:
these methods are no longer necessary, because the `error` parameter will have the information to determine if a user cancelled.